deltacat/utils/resources.py

```diff
@@ -172,3 +178,56 @@ def get_size_of_object_in_bytes(obj: object) -> float:
     if isinstance(obj, (list, tuple, set, frozenset)):
         return size + sum(map(get_size_of_object_in_bytes, obj))
     return size
+
+
+class NodeUtilizationOverTimeRange(AbstractContextManager):
```

Comment:
While I like this idea, would be fairly easy to have the syslog and other such EC2 logs in CloudWatch by modifying the cloudwatch agent config in parameter store: https://docs.aws.amazon.com/managedservices/latest/userguide/access-to-logs-ec2.html

Reply:
Interesting, so you're saying emit a new log group for each job?

Reply:
Yeah, correct. That log will print node statistics periodically.